tests/readmes/test_dispenser_flow.py

```diff
@@ -0,0 +1,88 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import os
+
+from ocean_lib.example_config import ExampleConfig
+from ocean_lib.models.dispenser import Dispenser
+from ocean_lib.models.models_structures import ErcCreateData, DispenserData
+from ocean_lib.ocean.mint_fake_ocean import mint_fake_OCEAN
+from ocean_lib.ocean.ocean import Ocean
+from ocean_lib.ocean.util import get_address_of_type
+from ocean_lib.web3_internal.constants import ZERO_ADDRESS
+from ocean_lib.web3_internal.currency import to_wei
+from ocean_lib.web3_internal.wallet import Wallet
+
+
+def test_dispenser_flow():
+    config = ExampleConfig.get_config()
+    ocean = Ocean(config)
+
+    # Create Alice's wallet
+    alice_private_key = os.getenv("TEST_PRIVATE_KEY1")
+    alice_wallet = Wallet(
+        ocean.web3,
+        alice_private_key,
+        config.block_confirmations,
+        config.transaction_timeout,
+    )
+    assert alice_wallet.address
+
+    # Mint OCEAN
+    mint_fake_OCEAN(config)
+    assert alice_wallet.web3.eth.get_balance(alice_wallet.address) > 0, "need ETH"
+
+    # Publish an NFT token
+    erc721_token = ocean.create_nft_token("NFTToken1", "NFT1", alice_wallet)
+    token_address = erc721_token.address
+    assert token_address
+    assert erc721_token.token_name() == "NFTToken1"
+    assert erc721_token.symbol() == "NFT1"
+
+    # Prepare data for ERC20 token
+    cap = to_wei(100)
```

Comment:
@mariacarmina please create an issue for the handling of to_wei vs web3.eth.toWei consistently. It seems in some places we are using David's utility, while in others we use the web3 one. No need to do it as part of this task, but please create a ticket and link it here.

Reply:
Done. #662